lib/main.js

```diff
@@ -2,11 +2,11 @@
 /*globals module require*/
 var exec = require('child_process').exec
   , fs   = require("fs")
-  , path = require("path")
-  , util = require("util");
+  , path = require("path");
 
-function puts(error, stdout/*, stderr*/) {
-  util.puts(stdout);
+function consolelog(error, stdout, stderr) {
+  console.log(stdout);
+  stderr ? console.log(stderr) : null;
```

Comment:
This ternary should be an `if` instead; that way there is no need for the `null` else block.